src/main/webapp/app/overview/exercise-details/course-exercise-details.component.ts

```diff
@@ -217,10 +206,11 @@ export class CourseExerciseDetailsComponent implements OnInit, OnDestroy {
             return this.sortedResults.find((result: Result) => !!result.completionDate);
         }
 
-        const currentResult = this.combinedParticipation.results.find((result: Result) => result.rated);
```

Comment:
This would mean that the first rated result is chosen as results are sorted from oldest to newest.

Reply:
You are right, good catch.